scripts/feeds

```diff
@@ -612,14 +612,13 @@ sub install_target_or_package {
 	my $name = shift;
 	my $force = shift;
 
-	my $this_feed_target = lookup_target($feed, $name);
-	$this_feed_target and do {
-		return install_target($this_feed_target, $name);
+	lookup_target($feed, $name) and do {
+		return install_target($feed, $name);
 	};
 
-	my $this_feed_src = lookup_src($feed, $name);
-	$this_feed_src and do {
-		return install_src($this_feed_src, $name, $force);
+	lookup_src($feed, $name) and do {
+		printf "there is src\n" if $name eq "zabbix";
```

Comment:
Do you seriously want to introduce a package name specific handling in the `feeds` script?

Reply:
Yeah, sorry, it is.